core/src/stored/stored.cc

```diff
@@ -253,6 +253,10 @@ int main(int argc, char* argv[])
   my_config = InitSdConfig(configfile, M_ERROR_TERM);
   ParseSdConfig(configfile, M_ERROR_TERM);
 
+  if (forge_on) {
+    my_config->AddWarning("Running with '-p' is for testing purposes only");
```

Comment:
How about "for testing and emergency recovery purposes only"

Reply:
agreed.